sympy/geometry/entity.py

```diff
@@ -171,7 +171,7 @@ def _repr_svg_(self):
             # will fall back to the next representation
             return None
 
-        if any([not x.is_number or not x.is_finite for x in bounds]):
+        if not all(x.is_number and x.is_finite for x in bounds):
             return None
```

Comment:
Does this change have an observable effect in the case that `is_finite` is None?

Maybe there could be a test for that...

Reply:
It may indeed be, but since this is printing code, I would assume that we will get some feedback pretty quickly... Right now, I do not really understand the code well enough to try to test it.